cogs/fun.py

```
@@ -0,0 +1,299 @@
+                    embed = discord.Embed(
+                        title=data['title'],
+                        url=data['postLink'],
+                        color=self.bot.config.PRIMARY_COLOR
+                    )
+                    embed.set_image(url=data['url'])
+                    embed.set_footer(text=f"👍 {data['ups']} | r/{data['subreddit']}")
+                    
+                    await ctx.send(embed=embed)
+                else:
+                    await ctx.send("❌ Couldn't fetch a meme right now!")
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)